dkg/pedersen/types/messages.go

```diff
@@ -150,6 +225,43 @@ func (d Deal) Serialize(ctx serde.Context) ([]byte, error) {
 	return data, nil
 }
 
+// deal messages for resharing process
```

Comment:
Comments must start with the element's name:

```suggestion
// DealResharing messages for resharing process
```